package.json

```diff
@@ -78,16 +72,31 @@
     "@react-three/fiber": "^8.9.2",
     "@types/debounce": "^1.2.1",
     "@types/react": "^18.0.25",
-    "microbundle": "^0.15.1",
+    "@types/three": "^0.149.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "three": "^0.144.0"
+    "three": "^0.144.0",
+    "vite": "^4.1.4"
   },
   "peerDependencies": {
     "@react-three/drei": ">=9.0.0",
     "@react-three/fiber": ">=8.0.0",
     "react": ">=18.0",
     "react-dom": ">=18.0",
     "three": ">=0.139.0"
+  },
+  "peerDependenciesMeta": {
+    "react": {
+      "optional": true
+    },
+    "react-dom": {
+      "optional": true
+    },
```

Comment:
React/ReactDOM is never optional.

But we should maybe add `three` as it is optional when importing from the `@14islands/r3f-scroll-rig/scrollbar` target